site/en/guide/gpu.ipynb

```diff
@@ -397,10 +397,10 @@
         "if gpus:\n",
         "  # Create 2 virtual GPUs with 1GB memory each\n",
         "  try:\n",
+        "   for i in range(len(gpus)):",
```

Comment:
The previous "LogicalDeviceConfiguration" was to limit the memory. This one is to simulate having multiple GPUs. If you actually have multiple GPUs, you don't need this. If you have just one, you want multiple LogicalDevices pinned to is. 

So I son't think this change aligns with what we're trying to do here.